example/iam/cloudformation.template.yml

```diff
@@ -0,0 +1,5 @@
+# Resources:
```

Comment:
Is this needed, or can it be removed?

Reply:
Idk why I check in things that are not supposed to be checked in. I will remove, thanks!

Reply:
Removed.